### PR TITLE
Add support for Symfony 6.0

### PR DESCRIPTION
**Pull request by 0lukas**
| Q               | A
| --------------- | -----
| Branch?         | master
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | -
| License         | MIT

### CODE REVIEW

File: composer.json
Comment (loic425) [inline composer.json:42]:
```suggestion
        "vimeo/psalm": "^4.23",
```

**Review by loic425 — CHANGES_REQUESTED** [review]

**Comment by loic425** [comment]
@0lukas Could you fix this issue?
https://github.com/Sylius/SyliusFixturesBundle/runs/6485051267?check_suite_focus=true#step:7:25

File: tests/Loader/FixtureLoaderTest.php
Comment (loic425) [inline tests/Loader/FixtureLoaderTest.php:96]:
```suggestion
        $result = $connection->fetchAssociative('SELECT count(*) as amount FROM testTable WHERE test_column = "test";');
        $this->assertSame(1, (int) $result['amount']);
```

File: tests/Loader/FixtureLoaderTest.php
Comment (loic425) [inline tests/Loader/FixtureLoaderTest.php:115]:
```suggestion
        $result = $connection->fetchAssociative('SELECT count(*) as amount FROM testTable WHERE test_column = "test";');
        $this->assertSame(1, (int) $result['amount']);
```

File: tests/Loader/FixtureLoaderTest.php
Comment (loic425) [inline tests/Loader/FixtureLoaderTest.php:158]:
```suggestion
        $result = $connection->fetchAssociative('SELECT count(*) as amount FROM testTable WHERE test_column = "test";');
        $this->assertSame(2, (int) $result['amount']);
```

**Comment by loic425** [comment]
@0lukas We need to check DBAL version on CI to check both DBAL 2 and 3 on Symfony 5..

**Comment by loic425** [comment]
@0lukas I'm suprised that it works with both Symfony 6 and DBAL 2. Maybe there is a mistake in the test.

**Review by loic425 — APPROVED** [review]

**Comment by 0lukas** [comment]
I'm not sure why it shouldn't work. Maybe you have specific case that i should check and write tests for?

**Comment by loic425** [comment]
> I'm not sure why it shouldn't work. Maybe you have specific case that i should check and write tests for?

I checked and everything is ok. We need a second review from @lchrusciel or @Zales0123, but the PR is ok for me.

File: .github/workflows/build.yml
Comment (lchrusciel) [inline .github/workflows/build.yml:21]: As we are testing these two values, it would be good to make a clear contract in our composer.json don't you think?

**Review by lchrusciel — APPROVED** [review]

**Pull request merged**
